src/server/zset_family.cc

```diff
@@ -2629,6 +2629,81 @@ void ZSetFamily::ZUnionStore(CmdArgList args, const CommandContext& cmd_cntx) {
   ZBooleanOperation(args, "zunionstore", true, true, cmd_cntx.tx, cmd_cntx.rb);
 }
 
+std::vector<ScoredMemberView> ZDiffInternal(CmdArgList args, const CommandContext& cmd_cntx,
+                                            size_t* maps_size_ptr = nullptr) {
+  vector<vector<ScoredMap>> maps(shard_set->size());
+
+  auto cb = [&](Transaction* t, EngineShard* shard) {
+    maps[shard->shard_id()] = OpFetch(shard, t);
```

Comment:
do you think that opfetch in this flow will / will not fetch the destination key of the zdiffstore command?